package.json

```diff
@@ -30,13 +28,21 @@
   "devDependencies": {
     "babel": "^5.5.8",
     "babel-eslint": "^3.1.15",
+    "babel-loader": "^5.1.4",
     "dependency-check": "^2.4.0",
     "eslint": "^0.23.0",
-    "faucet": "0.0.1",
-    "istanbul": "^0.3.15",
+    "isparta-loader": "^0.2.0",
+    "jasmine": "^2.3.1",
```

Comment:
Just for better understanding - what do you find bad in Jasmine?

I myself chose to use mocha + chai before, but then switched to jasmine, because it simply had more tooling around and used by all my peers, so it became easier to talk. So I have no strong opinion pro/against any of them.